yellowbrick/model_selection/validation_curve.py

```diff
@@ -0,0 +1,364 @@
+
+    This helper function is a wrapper to use the ValidationCurve in a fast,
+    visual analysis.
+
+    Parameters
+    ----------
+    model : a scikit-learn estimator
+        An object that implements ``fit`` and ``predict``, can be a
+        classifier, regressor, or clusterer so long as there is also a valid
+        associated scoring metric.
+
+        Note that the object is cloned for each validation.
+
+    X : array-like, shape (n_samples, n_features)
+        Training vector, where n_samples is the number of samples and
+        n_features is the number of features.
+
+    y : array-like, shape (n_samples) or (n_samples, n_features), optional
+        Target relative to X for classification or regression;
+        None for unsupervised learning.
+
+    param_name : string
+        Name of the parameter that will be varied.
+
+    param_range : array-like, shape (n_values,)
+        The values of the parameter that will be evaluated.
+
+    ax : matplotlib.Axes object, optional
+        The axes object to plot the figure on.
+
+    logx : boolean, optional
+        If True, plots the x-axis with a logarithmic scale.
+
+    groups : array-like, with shape (n_samples,)
+        Optional group labels for the samples used while splitting the dataset
+        into train/test sets.
+
+    cv : int, cross-validation generator or an iterable, optional
+        Determines the cross-validation splitting strategy.
+        Possible inputs for cv are:
+
+          - None, to use the default 3-fold cross-validation,
+          - integer, to specify the number of folds.
+          - An object to be used as a cross-validation generator.
+          - An iterable yielding train/test splits.
+
+        see the scikit-learn
+        `cross-validation guide <http://scikit-learn.org/stable/modules/cross_validation.html>`_
+        for more information on the possible strategies that can be used here.
+
+    scoring : string, callable or None, optional, default: None
+        A string or scorer callable object / function with signature
+        ``scorer(estimator, X, y)``. See scikit-learn model evaluation
+        documentation for names of possible metrics.
+
+    n_jobs : integer, optional
+        Number of jobs to run in parallel (default 1).
+
+    pre_dispatch : integer or string, optional
+        Number of predispatched jobs for parallel execution (default is
+        all). The option can reduce the allocated memory. The string can
+        be an expression like '2*n_jobs'.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers. These arguments are
+        also passed to the `poof()` method, e.g. can pass a path to save the
+        figure to.
+
+    Returns
+    -------
+    ax : matplotlib.Axes
+        The axes object that the validation curves were drawn on.
+    """
+
+    # Initialize the visualizer
+    oz = ValidationCurve(
```

Comment:
is `oz` the new convention for `viz`? I do like this

Reply:
I'm not sure for documentation purposes, but I'm certainly going to start using it in my code -- I hope you do too!

Reply:
agreed! unfortunately, there is no github emoji for:


![8-1-9-5cm-flying-monkey-vinyl-car-stickers-motorcycle-decals-window-car-accessories-c2-0607 jpg_640x640](https://user-images.githubusercontent.com/2944777/38269066-ac2855b2-374d-11e8-9304-9a4ef7f5e322.jpg)

Reply:
Hey there @Kautumn06 -- any updates on your progress with establishing a naming convention for instantiated visualizers in our code base and in the yb docs? Maybe you could open a new pull request so we can keep track of the discussion?